src/schemas/40-10007/schema.js

```diff
@@ -124,10 +125,14 @@ const emailFormat = {
   format: 'email'
 };
 
-definitions.address.required = ['street'];
+definitions.address.required = ['street', 'city', 'state', 'postalCode'];
```

Comment:
Will create a new frontend ticket for this one too.  Currently on the frontend, if the user selects a country other than  'USA' or 'Canada', the `state` field disappears.  Submission will fail.  `state` is required no matter what country is selected.

Reply:
Actually, I think this is a backend issue.  If no state is present, the backend should populate it with a default value.  Thoughts?  Thoughts on what that default value should be?
I'm going to email EOAS stakeholders too.

Reply:
Good question.  I didn't think about it because we currently leave the postalCode on the form as a required field regardless of country selected.

Reply:
Good point. Thanks.